Invalid POM definitions: missing `dependencies.dependency.version`

Running a maven dependency tree reveals the following for 1.9.0:

```
[WARNING] The POM for com.yubico:webauthn-server-core:jar:1.9.0 is invalid, transitive dependencies (if any) will not be available: 1 problem was encountered while building the effective model for com.yubico:webauthn-server-core:1.9.0
[ERROR] 'dependencies.dependency.version' for org.bouncycastle:bcprov-jdk15on:jar is missing. @ 

[WARNING] The POM for com.yubico:webauthn-server-attestation:jar:1.9.0 is invalid, transitive dependencies (if any) will not be available: 4 problems were encountered while building the effective model for com.yubico:webauthn-server-attestation:1.9.0
[ERROR] 'dependencies.dependency.version' for com.google.guava:guava:jar is missing. @ 
[ERROR] 'dependencies.dependency.version' for com.fasterxml.jackson.core:jackson-databind:jar is missing. @ 
[ERROR] 'dependencies.dependency.version' for org.bouncycastle:bcprov-jdk15on:jar is missing. @ 
[ERROR] 'dependencies.dependency.version' for org.slf4j:slf4j-api:jar is missing. @ 

[WARNING] The POM for com.yubico:yubico-util:jar:1.9.0 is invalid, transitive dependencies (if any) will not be available: 2 problems were encountered while building the effective model for com.yubico:yubico-util:1.9.0
[ERROR] 'dependencies.dependency.version' for com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:jar is missing. @ 
[ERROR] 'dependencies.dependency.version' for com.fasterxml.jackson.datatype:jackson-datatype-jdk8:jar is missing. @ 
```

For example, the pom that is published here is invalid:
https://search.maven.org/artifact/com.yubico/webauthn-server-core/1.9.0/pom

It presents:

```xml
    <dependency>
      <groupId>org.bouncycastle</groupId>
      <artifactId>bcprov-jdk15on</artifactId>
      <scope>runtime</scope>
    </dependency>
```


...without a version. Similar entries/pom for the others.

Maven details with JDK 11:

```
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
OS name: "mac os x", version: "10.16", arch: "x86_64", family: "mac"
```

--- a/NEWS
+++ b/NEWS
@@ -1,3 +1,10 @@
+== Version 1.9.1 (unreleased) ==
+
+webauthn-server-core:
+
+* Added missing `<dependencyManagement>` declaration to `webauthn-server-core` POM.
+
+
 == Version 1.9.0 ==
 
 webauthn-server-attestation:

--- a/webauthn-server-core-bundle/build.gradle
+++ b/webauthn-server-core-bundle/build.gradle
@@ -10,6 +10,8 @@ sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
+  api(platform(rootProject))
+
   api(
     project(':webauthn-server-core-minimal'),
   )

